anise/build.rs

```diff
@@ -1,13 +1,15 @@
-#[cfg(feature = "embed_ephem")]
-fn embed_ephem() {
+#[cfg(doc)]
+fn main() {}
+
+#[cfg(not(doc))]
```

Comment:
I think we only want to download if the embed feature is enabled and we aren't building the docs.

Reply:
sorry you're right, let me correct this